external/fv3fit/tests/emulation/test_layers_norm.py

```diff
@@ -0,0 +1,81 @@
+import pytest
+import numpy as np
+import tensorflow as tf
+
+from fv3fit.emulation import layers
+
+_all_layers = [
+    layers.StandardNormLayer,
+    layers.StandardDenormLayer,
+    layers.MaxFeatureStdNormLayer,
+    layers.MaxFeatureStdNormLayer,
+]
+
+
+@pytest.fixture(params=_all_layers)
+def layer_cls(request):
+    return request.param
+
+
+@pytest.fixture
+def tensor():
```

Comment:
Can you this simple data into a function `_get_tensor`, or maybe just copy paste the one-liner into each test function? I've spent a long time unwinding fixture complexity, so this makes me nervous.